### DISCUSSION TITLE
macOS Support?

### DISCUSSION THREAD

**Discussion by neodave** [guoyingtao/Mantis#149] [Q&A] [answered]
Hi,
I am really impressed with your implementation and appreciate you open sourcing it. I did have a question in regards to macOS support. The package file and readme says macOS is supported, but I believe that has to be Catalyst only. UIKit is used throughout the project so using this in a pure SwiftUI macOS app does not seem possible. Am I misreading the situation?
Cheers!

**Comment by guoyingtao** [comment, marked as answer]
You are right. I will update the README to descript it more accurately.

**Reply by neodave** [reply]
Thank you. The good news is that it does run in Catalyst. Will let you know if I find and or fix anything!

**Reply by rickshane** [reply]
In Catalyst, does the UI resize when the window resizes?

**Reply by guoyingtao** [reply]
If you present CropViewController with a fullScreen mode, the UI will resize when the window resizes. Also you may need to comment `use_frameworks!` in the Podfile if you uses CocoaPods and it crashes because `dyld: Library not loaded`
